pysqldb3/geopackage.py

```diff
@@ -201,69 +199,62 @@ def shp_to_gpkg(self, shp_name, gpkg_tbl = None, print_cmd=False):
         # check if the gpkg already exists. if so, we add another layer to the gpkg
         if not os.path.isfile(os.path.join(self.path, self.gpkg_name)):
```

Comment:
i think you should just create some methods that check if the geopackage exists and if a geopackage table exists, so you can just call that each time rather than reusing this code